### DISCUSSION TITLE
请求调整为默认开启域名前置使用内置Host

### DISCUSSION THREAD

**Discussion by RMWW** [AdNotFound/EhViewer#106] [Ideas]
这款白E可以直连，很不错，但需要自行设置开启域名前置启动内置Host才行，否则无法使用Cookies直连登陆。请求调整为默认开启域名前置使用内置Host，以方便可以使用Cookies直连登陆。

**Comment by AdNotFound** [comment]
0642635 已解决

**Reply by RMWW** [reply]
感谢现在这款白E也能通过Cookies直连登陆了，还差最后两个细节就完美了，将在下个讨论环节中提出。
